### PR TITLE
Windows: Support for stats (docker #25737)

### PR DESCRIPTION
**Pull request by jhowardmsft**
Signed-off-by: John Howard <jhoward@microsoft.com>

Adds support for docker stats on Windows. See https://github.com/docker/docker/pull/25737 for detailed information.

### CODE REVIEW

**Comment by jhowardmsft** [comment]
ping @docker/core-engine-api-maintainers

File: client/image_build.go [hunk 1]
```diff
@@ -113,7 +113,8 @@ func imageBuildOptionsToQuery(options types.ImageBuildOptions) (url.Values, erro
 	return query, nil
 }
 
-func getDockerOS(serverHeader string) string {
+// GetDockerOS returns the operating system based on the server header from the daemon.
+func GetDockerOS(serverHeader string) string {
```
Comment (vdemeester) [inline client/image_build.go]: We might want to move this function out of this file but that can be done in a follow-up PR :wink:

**Comment by vdemeester** [comment]
LGTM 🐮

File: client/image_build.go (hunk 1 above)
Comment (jhowardmsft) [inline client/image_build.go]: Yup, make sense. I'll make sure to do that as a follow-up.

**Comment by jhowardmsft** [comment]
The matching docker PR https://github.com/docker/docker/pull/25737 is all green :green_heart: now and just waiting for this to be merged so it can be rebased.

**Comment by jhowardmsft** [comment]
Closing now engine-api is back in docker.

**Pull request closed without merging**